### ISSUE TITLE
Do you still need commit access?

### ISSUE THREAD

**Issue by tstellar** [llvm/llvm-project#91249]
### TLDR: If you want to retain your commit access, please comment on this issue.  Otherwise, you can unsubscribe from this issue and ignore it.  Commit access is not required to contribute to the project.  You can still create Pull Requests without commit access.

@eywdck2l @gbreynoo @pzhengqc @wolfy1961 @uabjean @rochauha @rcraik @etiotto @mitchell-stellar @SEmmmer @dcandler @mdtrent @romanova-ekaterina @djtodoro @dnuzman @henrickarlsson @rlougher @dpreobra @kbelochapka @drichards-87

LLVM has a policy of downgrading write access to its repositories for accounts with long term inactivity. This is done because inactive accounts with high levels of access tend to be at increased risk of compromise and this is one tactic that the project employs to guard itself from malicious actors.  Note that write access is not required to contribute to the project.  You can still submit pull requests and have someone else merge them.

Our project policy is to ping anyone with less than five 'interactions' with the repositories over a 12 month period to see if they still need commit access.  An 'interaction' and be any one of:

* Pushing a commit.
* Merging a pull request (either their own or someone else’s).
* Commenting on a PR.

If you want to retain your commit access, please post a comment on this issue.  If you do not want to keep your commit access, you can just ignore this issue.  If you have not responded in 6 weeks, then you will move moved from the 'write' role within the project to the 'triage' role.  The 'triage' role is still a privileged role and will allow you to do the following:

* Review Pull Requests.
* Comment on issues.
* Apply/dismiss labels.
* Close, reopen, and assign all issues and pull requests.
* Apply milestones.
* Mark duplicate issues and pull requests.
* Request pull request reviews.
* Hide anyone’s comments.

**Comment by wolfy1961** [comment]
I would like to keep my commit access. Thanks.

**Comment by mdtrent** [comment]
I would like to retain commit access.

**Comment by pzhengqc** [comment]
I'd like to keep my commit access. Thanks.

**Comment by dpreobra** [comment]
I’d like to keep my commit access. 
Thanks.

**Comment by kbelochapka** [comment]
I would like to keep my commit access.

Thanks, Konstantin


From: Tom Stellard ***@***.***>
Sent: Monday, May 6, 2024 11:11 AM
To: llvm/llvm-project ***@***.***>
Cc: Belochapka, Konstantin ***@***.***>; Mention ***@***.***>
Subject: [llvm/llvm-project] Do you still need commit access? (Issue #91249)

TLDR: If you want to retain your commit access, please comment on this issue. Otherwise, you can unsubscribe from this issue and ignore it. Commit access is not required to contribute to the project. You can still create Pull Requests without commit access.

@eywdck2l<https://github.com/eywdck2l> @gbreynoo<https://github.com/gbreynoo> @pzhengqc<https://github.com/pzhengqc> @wolfy1961<https://github.com/wolfy1961> @uabjean<https://github.com/uabjean> @rochauha<https://github.com/rochauha> @rcraik<https://github.com/rcraik> @etiotto<https://github.com/etiotto> @mitchell-stellar<https://github.com/mitchell-stellar> @SEmmmer<https://github.com/SEmmmer> @dcandler<https://github.com/dcandler> @mdtrent<https://github.com/mdtrent> @romanova-ekaterina<https://github.com/romanova-ekaterina> @djtodoro<https://github.com/djtodoro> @dnuzman<https://github.com/dnuzman> @henrickarlsson<https://github.com/henrickarlsson> @rlougher<https://github.com/rlougher> @dpreobra<https://github.com/dpreobra> @kbelochapka<https://github.com/kbelochapka> @drichards-87<https://github.com/drichards-87>

LLVM has a policy of downgrading write access to its repositories for accounts with long term inactivity. This is done because inactive accounts with high levels of access tend to be at increased risk of compromise and this is one tactic that the project employs to guard itself from malicious actors. Note that write access is not required to contribute to the project. You can still submit pull requests and have someone else merge them.

Our project policy is to ping anyone with less than five 'interactions' with the repositories over a 12 month period to see if they still need commit access. An 'interaction' and be any one of:

  *   Pushing a commit.
  *   Merging a pull request (either their own or someone else’s).
  *   Commenting on a PR.

If you want to retain your commit access, please post a comment on this issue. If you do not want to keep your commit access, you can just ignore this issue. If you have not responded in 6 weeks, then you will move moved from the 'write' role within the project to the 'triage' role. The 'triage' role is still a privileged role and will allow you to do the following:

  *   Review Pull Requests.
  *   Comment on issues.
  *   Apply/dismiss labels.
  *   Close, reopen, and assign all issues and pull requests.
  *   Apply milestones.
  *   Mark duplicate issues and pull requests.
  *   Request pull request reviews.
  *   Hide anyone’s comments.

**Comment by dnuzman** [comment]
I would like to keep my commit access. Thanks.

**Comment by dcandler** [comment]
I'd like to retain commit access. Thanks!

**Comment by romanova-ekaterina** [comment]
I would like to keep my commit access.
Thank you!

From: Tom Stellard ***@***.***>
Sent: Monday, May 6, 2024 11:11 AM
To: llvm/llvm-project ***@***.***>
Cc: Romanova, Katya ***@***.***>; Mention ***@***.***>
Subject: [llvm/llvm-project] Do you still need commit access? (Issue #91249)

TLDR: If you want to retain your commit access, please comment on this issue. Otherwise, you can unsubscribe from this issue and ignore it. Commit access is not required to contribute to the project. You can still create Pull Requests without commit access.

@eywdck2l<https://github.com/eywdck2l> @gbreynoo<https://github.com/gbreynoo> @pzhengqc<https://github.com/pzhengqc> @wolfy1961<https://github.com/wolfy1961> @uabjean<https://github.com/uabjean> @rochauha<https://github.com/rochauha> @rcraik<https://github.com/rcraik> @etiotto<https://github.com/etiotto> @mitchell-stellar<https://github.com/mitchell-stellar> @SEmmmer<https://github.com/SEmmmer> @dcandler<https://github.com/dcandler> @mdtrent<https://github.com/mdtrent> @romanova-ekaterina<https://github.com/romanova-ekaterina> @djtodoro<https://github.com/djtodoro> @dnuzman<https://github.com/dnuzman> @henrickarlsson<https://github.com/henrickarlsson> @rlougher<https://github.com/rlougher> @dpreobra<https://github.com/dpreobra> @kbelochapka<https://github.com/kbelochapka> @drichards-87<https://github.com/drichards-87>

LLVM has a policy of downgrading write access to its repositories for accounts with long term inactivity. This is done because inactive accounts with high levels of access tend to be at increased risk of compromise and this is one tactic that the project employs to guard itself from malicious actors. Note that write access is not required to contribute to the project. You can still submit pull requests and have someone else merge them.

Our project policy is to ping anyone with less than five 'interactions' with the repositories over a 12 month period to see if they still need commit access. An 'interaction' and be any one of:

  *   Pushing a commit.
  *   Merging a pull request (either their own or someone else’s).
  *   Commenting on a PR.

If you want to retain your commit access, please post a comment on this issue. If you do not want to keep your commit access, you can just ignore this issue. If you have not responded in 6 weeks, then you will move moved from the 'write' role within the project to the 'triage' role. The 'triage' role is still a privileged role and will allow you to do the following:

  *   Review Pull Requests.
  *   Comment on issues.
  *   Apply/dismiss labels.
  *   Close, reopen, and assign all issues and pull requests.
  *   Apply milestones.
  *   Mark duplicate issues and pull requests.
  *   Request pull request reviews.
  *   Hide anyone’s comments.

**Comment by rlougher** [comment]
I would like to retain commit access. Thanks!

**Comment by tstellar** [comment]
@eywdck2l @gbreynoo @uabjean @rochauha @rcraik @etiotto @mitchell-stellar @SEmmmer @djtodoro @henrickarlsson @drichards-87 Reminder, if you want to retain your commit access, please comment on this ticket in the next week.

**Comment by kbelochapka** [comment]
I would like to retain commit access.
Thanks!

**Comment by djtodoro** [comment]
I would like to retain commit access. Thanks a lot!

**Comment by romanova-ekaterina** [comment]
I would like to keep my commit access. Thank you!
Katya.

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by tstellar** [comment]
@eywdck2l @gbreynoo @uabjean @rochauha @rcraik @etiotto @mitchell-stellar @SEmmmer @henrickarlsson @drichards-87 added to the LLVM Triagers team. If you need write access again, please follow these instructions: https://llvm.org/docs/DeveloperPolicy.html#obtaining-commit-access
